packages/core/src/components/table/table-body-row-expandable/table-body-row-expandable.tsx

```diff
@@ -204,8 +227,25 @@ export class TdsTableBodyRowExpandable {
           }}
           part="expand-row"
         >
-          <td class="tds-table__cell-expand" colSpan={this.columnsNumber}>
-            <slot name="expand-row" />
+          <td
+            class={isFlex ? 'tds-table__cell-expandflex' : 'tds-table__cell-expand'}
+            colSpan={this.columnsNumber}
+            style={cellStyle}
+          >
+            <div
+              class={{
+                'flex-container': isFlex,
+              }}
+              style={{
```

Comment:
What do you think about refactoring this part for readability reason? Maybe something like this. 
```suggestion
              const flexStyles = isFlex
  ? {
      flexDirection: this.flexDirection,
      flexWrap: 'wrap',
      width: '100%',
      justifyContent: this.justifyContent,
      alignItems: this.alignItems,
    }
  : {};

return (
  <div style={flexStyles}>
    {/* Your component content */}
  </div>
);
```